### ISSUE TITLE
Videos are not completely fullscreen

### ISSUE THREAD

**Issue by PrinceRaj53** [TeamNewPipe/NewPipe#8774]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.23.2

### Steps to reproduce the bug

1. Play any video
2. Switch to full screen
3. Touch the arrow on top right corner
4. Set the "zoom" option
5. You'll see that the video is still not completely full screen & there is still space left on the left side.

### Expected behavior

The videos should be played full screen.

### Actual behavior

The videos are not in fullscreen.

### Screenshots/Screen recordings

This is how it should've worked.

![Should be like this](https://user-images.githubusercontent.com/96355726/184500779-d9898a26-da38-429a-b263-a42db01fb2b4.jpg)

And this is how it looks right now :

![Not fullscreen](https://user-images.githubusercontent.com/96355726/184500816-c26da398-280c-4ea1-adec-54b86251075e.jpg)

### Logs

None.

### Affected Android/Custom ROM version

android 12

### Affected device model

Realme GT Neo 2 (RMX3370)

### Additional information

Interestingly, the debug apk that was provided just before 0.23.2 actually shows the video in full screen, edge to edge, without any issues. 

The apk that worked & I was talking about is this one : https://github.com/TeamNewPipe/NewPipeExtractor/pull/891 This is the only one where the video was completely fullscreen.

**Comment by AudricV** [comment]
Was it reproducible on 0.23.1? If so, then it may have been fixed already fixed on the dev version, as the APK should be based on the dev branch (could you confirm, please @Theta-Dev?).

**Comment by PrinceRaj53** [comment]
> Was it reproducible on 0.23.1? If so, then it may have been fixed already fixed on the dev version, as the APK should be based on the dev branch (could you confirm, please @Theta-Dev?).

Well yes, this issue was in 0.23.1 as well. 

Like I said, the problem is still present in 0.23.2. Only in the debug apk that I have mentioned above, the videos were in complete fullscreen.

**Comment by PrinceRaj53** [comment]
Sorry, misclicked on "close with comment" option.

**Comment by PrinceRaj53** [comment]
Can someone please reopen the issue? i mistakenly closed it.

**Comment by opusforlife2** [comment]
If you closed it, you can open it.

**Comment by PrinceRaj53** [comment]
I have been trying to reopen this issue but it remains closed.

**Comment by PrinceRaj53** [comment]
So as I said before, the bug was present in the previous version as well as the new version. Only in the debug app it works without issues. The first screenshot("this is how it should've worked") was taken in the debug app.

**Comment by goat-oss** [comment]
Same error as above on Android 12.
Samsung Phone.

**Comment by opusforlife2** [comment]
> Interestingly, the debug apk that was provided just before 0.23.2 actually shows the video in full screen, edge to edge, without any issues.

It is possible that that debug APK was built with the latest changes of not just the extractor, but also the app, which contains a lot of player improvements. To test this, see if the next full release fixes your issue.

**Comment by PrinceRaj53** [comment]
> > Interestingly, the debug apk that was provided just before 0.23.2 actually shows the video in full screen, edge to edge, without any issues.
> 
> It is possible that that debug APK was built with the latest changes of not just the extractor, but also the app, which contains a lot of player improvements. To test this, see if the next full release fixes your issue.

You mean the debug apk is based on dev branch? I see. If only thetadev could confirm it. I'm not sure if I'm allowed to ping him.

Hopefully it does get fixed in the next release, honestly its kind of frustrating to watch videos when they are clipped from one side or from both sides(sometimes its cut from both sides).

For now it seems I'll have to keep using the debug app instead of the full release.

**Comment by opusforlife2** [comment]
> I'm not sure if I'm allowed to ping him.

If you're polite about it, it's perfectly fine. People usually have problems if you ping them despite their wishes, or mass ping a bunch of people.

> when they are clipped from one side or from both sides(sometimes its cut from both sides).

Have you fiddled with Fit/Fill/Zoom? It's next to the subtitle language menu.

**Comment by PrinceRaj53** [comment]
> > when they are clipped from one side or from both sides(sometimes its cut from both sides).
> 
> Have you fiddled with Fit/Fill/Zoom? It's next to the subtitle language menu.

Yes. I always set it to zoom when i watch a video to make sure its completely fullscreen but it always has cuts on the sides. Only in the debug app it works as intended.

I hope @Theta-Dev doesn't mind me pinging him.

**Comment by AudricV** [comment]
Could you check if it is also fixed in the APK from this CI run or not?

https://github.com/TeamNewPipe/NewPipe/actions/runs/2856253829

**Comment by PrinceRaj53** [comment]
> Could you check if it is also fixed in the APK from this CI run or not?
> 
> https://github.com/TeamNewPipe/NewPipe/actions/runs/2856253829

Yes it is fixed. Just installed and checked. Even works better than previous debug apk somehow.

Does it mean this will be fixed in the next release?

**Comment by goat-oss** [comment]
![bcfc8d41-138b-4960-9c4c-851e50a93472](https://user-images.githubusercontent.com/88162555/186131126-9f8cd352-c678-4dff-819d-3dd0891f8d3f.png)
Enabling full screen shows the screen only half like the screenshot in landscape mode.

**Comment by AudricV** [comment]
> Now it cuts the screen from the top & bottom side instead of the left side.

Is it the same issue as #8727?

**Comment by opusforlife2** [comment]
@PrinceRaj53 If the debug build fixed this as well, then you just need to wait for the next major release.

**Comment by opusforlife2** [comment]
Update the description to match the current status.

**Comment by ChipsEatingPenguin** [comment]
I do have this issue as well just on a vastly different device

Samsung SM-P900
Android 5.0.2
Newpipe 0.25.1
When I choose to zoom the video to fill the screen (16:10 ratio) Right 1:10th of the screen gets a vertical black bar

**Comment by mtaras0** [comment]
The bug is still happening, on release versions and on the current 26 RC. My device is an old tablet with Android 5, no notch no nothing, Zoom does zoom the video, but shifts it around depending on the viewing mode -- in fullscreen it's shifted to the left with the black bar on the right, when not in fullscreen it's shifted to the top, bottom half of the player is black. Looks like the shift is going in the opposite direction, like the sign is flipped, the amount of black space is about how much video should've been cut off by being zoomed it
